src/app/containers/RadioSchedule/index.stories.jsx

```diff
@@ -10,12 +11,14 @@ import WithTimeMachine from '#testHelpers/withTimeMachine';
 
 // Currently, only these services have radio schedule data
 const validServices = [
+  'afrique',
```

Comment:
Afrique has been added to the list of valid services, but there isn't a corresponding update to data in `.storybook/static/data/afrique` and `data/afrique/bbc_afrique_radio`

Reply:
Seen thekp's comment above.